node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -402,12 +402,15 @@ void doConverge(NodeAgentContext context) {
         switch (node.state()) {
             case ready:
             case reserved:
-            case parked:
             case failed:
             case inactive:
                 removeContainerIfNeededUpdateContainerState(context, container);
                 updateNodeRepoWithCurrentAttributes(context);
                 break;
+            case parked:
+                updateNodeRepoWithCurrentAttributes(context);
+                stopServices(context);
```

Comment:
This will log "Stopping services" on every tick.

Reply:
Yeah, good point, will add a stopServicesIfNeeded() and use that instead